### ISSUE TITLE
False warning in `${{ secrets.<...> }}` validation

### ISSUE THREAD

**Issue by disrupted** [rhysd/actionlint#127]
In this minimal example of [docker/login-action](https://github.com/docker/login-action)

```yaml
name: Docker

on:
  workflow_call:

jobs:
  docker:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3

      - name: Login to DockerHub
        uses: docker/login-action@v1
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

actionlint is reporting:
```sh
.github/workflows/docker.yml:15:25: property "dockerhub_username" is not defined in object type {github_token: string} [expression]
   |
15 |           username: ${{ secrets.DOCKERHUB_USERNAME }}
   |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~
.github/workflows/docker.yml:16:25: property "dockerhub_token" is not defined in object type {github_token: string} [expression]
   |
16 |           password: ${{ secrets.DOCKERHUB_TOKEN }}
```

It looks to me that it's trying to validate secrets for GitHub instead
